creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -238,15 +238,68 @@ spec:
     pod: {}
 ----
 
+______________________________________________________________________________
+**Note:** If a specific MAC address is configured in the virtual machine
+interface spec, it's passed to underlying CNI plugin that should configure
+backend to allow for this particular address. While some plugins are known to
+work without special support for custom MAC addresses, others do (for example,
+OpenShift SDN needs to configure OVS flows to allow traffic meant for the
+chosen MAC address).
+______________________________________________________________________________
+
+______________________________________________________________________________
+**Note:** If a plugin doesn't support `mac` attribute defined by Kubernetes
+Network Plumbing Working Group, one can use `tuning` plugin that is one of
+reference plugins maintained by the group. This can be used as follows:
+
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ptp-mac
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "name": "ptp-mac",
+      "plugins": [
+        {
+          "type": "ptp",
+          "ipam": {
+            "type": "host-local",
+            "subnet": "10.1.1.0/24"
+          },
+        },
+        {
+          "type": "tuning",
+        }
+      ]
+    }'
+----
+
+
+This approach won't work for other plugins that needs special treatment for
+custom MAC addresses. For example, OpenShift SDN is not compatible with
+`tuning` plugin.
+
+Plugins that handle custom MAC addresses natively: `ovs`.
+
+Plugins that don't need special MAC address treatment: `sriov`.
```

Comment:
That means sriov can read "mac" parameter on its own or do we need to use tuning?

Reply:
For SR-IOV devices, since we use VFIO, we don't care b/c there is no netlink device once we pass it to guest.

Reply:
Mentioned I assume VFIO mode here (which is the mode we should use for kubevirt).